src/formatDate.js

```diff
@@ -50,7 +50,26 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateSplit = date.split(fromFormat[fromFormat.length - 1]);
+  const dateArr = [];
+
+  fromFormat.forEach((i, j) => {
+    dateArr[i] = dateSplit[j];
```

Comment:
`i` is some string in this case, for example `'DD'`. Do you use it as a key of ... array? You should have `dateObject` in this case